src/panels/lovelace/hui-root.ts

```diff
@@ -738,7 +778,7 @@ class HUIRoot extends LitElement {
           color: var(--error-color);
         }
         #view {
-          min-height: calc(100vh - 112px);
+          min-height: calc(100vh - var(--header-height));
```

Comment:
This is weird? 112px is not the same as header height ?

Reply:
112px was incorrect and header height is the correction.